.github/workflows/cypress.yml

```diff
@@ -80,3 +80,77 @@ jobs:
           CYPRESS_TEST_USER_EMAIL: "cypress@example.com"
           CYPRESS_TEST_USER_PASS: ${{ secrets.CYPRESS_TEST_USER_PASS }}
           CYPRESS_baseUrl: ${{ github.event.deployment_status.environment_url }}
+
+  configuremobile:
+    needs: Setup
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
@oudeismetis can you see what the issue with this is?
It says 
`fatal: bad revision 'origin/main'`